.github/workflows/reusable-wasi.yml

```diff
@@ -53,14 +53,16 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: '3.x'
+    - name: "Runner image version"
+      run: echo "IMAGE_OS_VERSION=${ImageOS}-${ImageVersion}" >> "$GITHUB_ENV"
```

Comment:
This wasn't being set in this job, so `env.IMAGE_VERSION` was evaluating as an empty string in the keys below.